src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts

```diff
@@ -111,6 +111,7 @@ export class HearingAttendanceComponent extends RequestHearingPageFlow implement
           ...partyDetailsModel,
           partyAmendmentStatus: AmendmentLabelStatus.ACTION_NEEDED
         };
+        (this.attendanceFormGroup.controls.parties as FormArray).push(this.patchValues(partyDetailsModel) as FormGroup);
```

Comment:
Can we add safety checks please
`const parties = this.attendanceFormGroup.controls.parties as FormArray;
 const  patchValues = this.patchValues(partyDetailsModel) as FormGroup;
 parties?.push(patchValues)...
`

Reply:
Thank you @udaydenduluri33, I checked the code, and `parties` is initialised as `fb.array([])` when initialising the `this.attendanceFormGroup`, and the `partyDetailsModel` always contain a value as it is inside the `for` loop.